README.md

```diff
@@ -227,22 +227,22 @@ bionemo-esm2-recipe \
 
 > ⚠️ **IMPORTANT:** Inspect and edit the contents of the outputted my_config.json as you see fit
 
-> NOTE: To pretrain from an existing checkpoint, simply pass in the path --initial-ckpt-path to the recipe command. This will populate the JSON with the correct field to ensure pretraining is initialized from an existing checkpoint.
+> NOTE: To pretrain from an existing checkpoint, simply pass in the path --initial-ckpt-path to the recipe command. This will populate the YAML with the correct field to ensure pretraining is initialized from an existing checkpoint.
 
 To submit a training job with the passed config, first update the json file with any additional execution parameters
 of your choosing: number of devices, workers, steps, etc. Second, invoke our training entrypoint. To do this, we need
 three things:
 
-- Configuration file, the JSON produced by the previous step
-- Model config type, in this case the pretraining config. This will validate the arguments in the config JSON against
+- Configuration file, the YAML produced by the previous step
+- Model config type, in this case the pretraining config. This will validate the arguments in the config YAML against
     those required for pretraining. Alternatively, things like fine-tuning with custom task heads may be specified here.
     This allows for mixing/matching Data Modules with various tasks.
 - Data Config type, this specifies how to parse, validate, and prepare the DataModule. This may change depending on task,
 for example, pretraining ESM2 uses a protein cluster oriented sampling method. In the case of inference or fine-tuning
 a pretrained model, a simple fasta file may be sufficient. There is a one-to-one relationship between DataConfig types
 and DataModule types.
 
-> ⚠️ **Warning:** This setup does NO configuration of Weights and Biases. Edit your config JSON and populate it with your WandB details.
+> ⚠️ **Warning:** This setup does NO configuration of Weights and Biases. Edit your config YAML and populate it with your WandB details.
 
 ```
 bionemo-esm2-train \
```

Comment:
my_config.json -> yaml

Reply:
what does it mean  "t" in `data-config-t`?

Reply:
is t refers to the class? maybe istead of t we can have cls?

Reply:
we could move it to `cls`, I dont have a preference.